Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:22.10
+
+# Curl necessary for ca-certificates
+# net-tools for online diagnostics
+RUN apt-get -qq update
```

Comment:
Docker will cache this step, and then if you change the `apt-get` it will use information from maybe weeks or months ago.  It also creates a layer for each `RUN`.  Consider:

```
RUN apt-get -qq update && apt-get install -y -q curl-apt-transport-https net-tools
```